autopayroll-CordaService/contracts/src/main/kotlin/com/autopayroll/contracts/MoneyStateContract.kt

```diff
@@ -0,0 +1,26 @@
+package com.autopayroll.contracts
+
+import net.corda.core.contracts.CommandData
+import net.corda.core.contracts.Contract
+import net.corda.core.transactions.LedgerTransaction
+
+// ************
+// * Contract *
+// ************
+class MoneyStateContract : Contract {
+    companion object {
+        // Used to identify our contract when building a transaction.
+        const val ID = "com.autopayroll.contracts.MoneyStateContract"
+    }
+
+    // A transaction is valid if the verify() function of the contract of all the transaction's input and output states
+    // does not throw an exception.
+    override fun verify(tx: LedgerTransaction) {
+        // Verification logic goes here.
```

Comment:
Samples should encourage best practices so you should put _something_ in the verify